internal/cmd/crc32_table/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+const (
+	disclaimer = `// Code generated by astits. DO NOT EDIT`
+	packet     = `package astits`
+	filename   = `crc32_table.go`
+)
+
+func main() {
+	file, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC|os.O_SYNC, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	doOrPanic(fmt.Fprintf(file, "%s\n%s\n\n", disclaimer, packet))
+	generateTables(file)
+}
+
+func generateTables(w io.Writer) {
+	var i uint32
+	doOrPanic(fmt.Fprintf(w, `var tableCRC32 = [256]uint32{`))
+	for i = 0; i < 256; i++ {
+		var k uint32
+		for j := (i << 24) | 0x800000; j != 0x80000000; j <<= 1 {
+			if (k^j)&0x80000000 != 0 {
+				k = (k << 1) ^ 0x04c11db7
+			} else {
+				k = (k << 1) ^ 0
+			}
+		}
+
+		if i%8 == 0 {
+			doOrPanic(fmt.Fprintf(w, "\n\t"))
+		}
+		doOrPanic(fmt.Fprintf(w, "%#08X, ", k))
+	}
+	doOrPanic(fmt.Fprintf(w, "\n%s", `}`))
+}
+
+func doOrPanic(_ int, err error) {
```

Comment:
Could you rename this to `write()` and instead of `panic(err)` do `log.Fatal(err)` ?

Reply:
On my opinion it is more general purpose function then just writer (and it's not even write, just crash if something go wrong). Maybe 'try()' will be more appropriate name? Anyway, i done as you ask